pkg/logs/auditor/auditor.go

```diff
@@ -87,7 +88,7 @@ func (a *Auditor) run() {
 				return
 			}
 			// update the registry with new entry
-			a.updateRegistry(msg.GetOrigin().Identifier, msg.GetOrigin().Offset, msg.GetOrigin().Timestamp)
+			a.updateRegistry(msg.GetOrigin().Identifier, msg.GetOrigin().Offset, msg.GetOrigin().Timestamp, msg.GetOrigin().Cursor)
```

Comment:
Question: Will the auditor api be clearer by adding sources like this (`Cursor`) or should we limit the number of fields here, by _renaming_ and _reusing_ `timestamp` which is already a string

Reply:
yes, maybe we could still accept ints to avoid conversions.
So would we want to do this now?